documentation-website/src/pages/quick-start/index.tsx

```diff
@@ -18,7 +18,7 @@ const Index = () => <Layout
     prefix={'quick-start.step2.description'}
     postfix={'quick-start.step2.result'}
     level={'h2'}
-    language={'bash'}>npx -p @idrinth/api-bench-cli iabmp</CodeUnit>
+    language={'bash'}>npx -p @idrinth-api-bench/api-bench-cli iabmp</CodeUnit>
```

Comment:
this package will never exist, please revert the change for now

Reply:
Reverted